packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -103,7 +103,9 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                                     id='dt_bt_label'
                                     icon={<Icon icon='IcContractPayout' size={24} />}
                                     label={localize('Payout per point')}
-                                    value={`${display_number_of_contracts} ${getCurrencyDisplayCode(currency)}` || ' - '}
+                                    value={
+                                        `${display_number_of_contracts} ${getCurrencyDisplayCode(currency)}` || ' - '
```

Comment:
@arshad-rao-deriv after turbos release, this alignment will be irrelevant. please don't include alignment-only changes unrelated to your PR scope.